core/src/main/java/nl/nn/adapterframework/webcontrol/api/AuthorizationFilter.java

```diff
@@ -49,62 +34,57 @@
 @Produces(MediaType.APPLICATION_JSON)
 public class AuthorizationFilter implements ContainerRequestFilter {
 
-	private static final ServerResponse ACCESS_FORBIDDEN = new ServerResponse(null, 403, new Headers<Object>());
-	private static final ServerResponse SERVER_ERROR = new ServerResponse(null, 500, new Headers<Object>());
-	protected Logger log = LogUtil.getLogger(this);
-
+	@Override
```

Comment:
If this file is no longer required, you can remove it.
Have you tested the authorizations on specific endpoints?

Reply:
no, i don't know how to test it... can you tell me how please.?